src/store/messagesStore.js

```diff
@@ -651,10 +652,12 @@ const actions = {
 				updatedMessage,
 			})
 			context.dispatch('processMessage', { token, message: response.data.ocs.data })
+			EventBus.emit('editing-message-processing', { messageId, value: false })
 		} catch (error) {
 			console.error(error)
 			// Restore the previous message state
 			context.commit('addMessage', { token, message })
+			EventBus.emit('editing-message-processing', { messageId, value: false })
```

Comment:
` finally {...}`

Reply:
error thrown in catch block doesn't allow to do that